ROADMAP.md

```diff
@@ -0,0 +1,13 @@
+# Lightning Roadmap
+
+This document defines the roadmap for Lightning development.
+
+## TiDB:
+
```

Comment:
Should we add a `workload supports` section? 

+ [ ] Workload supports
    - [ ] Support 1TiB single tables restore
    - [ ] Support 10TiB database